ui/admin/tests/e2e/pages/aliases.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef */
+const { expect } = require('@playwright/test');
+const { nanoid } = require('nanoid');
+
+class AliasesPage {
+  constructor(page) {
+    this.page = page;
+  }
+
+  /**
+   * Creates an alias
+   * @param {string} alias Value of the alias
+   * @param {string} targetId ID of the target
+   * @returns Name of the alias
+   */
+  async createAliasForTarget(alias, targetId) {
+    const aliasName = 'Alias ' + nanoid();
+
+    await this.page.getByRole('link', { name: 'Orgs', exact: true }).click();
+    await this.page.getByRole('link', { name: 'Aliases' }).click();
+    await expect(
+      this.page
+        .getByRole('navigation', { name: 'breadcrumbs' })
+        .getByText('Aliases'),
+    ).toBeVisible();
+
+    const newButtonIsVisible = await this.page
+      .getByRole('link', { name: 'Create a new alias', exact: true })
+      .isVisible();
+    if (newButtonIsVisible) {
+      await this.page
+        .getByRole('link', { name: 'Create a new alias', exact: true })
+        .click();
+    } else {
+      await this.page
+        .getByRole('link', { name: 'New Alias', exact: true })
+        .click();
+    }
+
+    await this.page.getByLabel('Name').fill(aliasName);
+    await this.page.getByLabel('Description').fill('This is an automated test');
+    await this.page.getByLabel('Alias Value').fill(alias);
+    await this.page.getByLabel('Target ID').fill(targetId);
+
+    await this.page.getByRole('button', { name: 'Save' }).click();
+    await expect(
+      this.page.getByRole('alert').getByText('Success', { exact: true }),
+    ).toBeVisible();
+    await this.page.getByRole('button', { name: 'Dismiss' }).click();
```

Comment:
Per the other comment about a `global.js` file, is it worth creating a helper method to dismiss these success alerts? (this is used all over)